x-pack/plugins/apm/typings/es_schemas/Span.ts

```diff
@@ -14,7 +15,7 @@ interface Processor {
 
 export interface Span extends APMDoc {
   processor: Processor;
-  service: { name: string };
+  service: Service;
```

Comment:
I think only transactions and errors have the other fields (`runtime`, `language` and `framework`). AFAICT `span` docs only have `service.name`

Reply:
It seems like

Reply:
nvm. I got confused because we access `service.agent` here:
https://github.com/elastic/kibana/blob/4749c6aab6ef39c82dd363b18dc9cc337e4059b2/x-pack/plugins/apm/public/components/shared/PropertiesTable/propertyConfig.ts#L61

I think @formgeist is fixing that in a separate PR though.

Reply:
Removed `agent` here https://github.com/elastic/kibana/pull/32199/commits/f991caaec8f55447a8b235a9fd74b9b446ee40e3